lib/configParser.js

```diff
@@ -95,7 +95,7 @@ ConfigParser.resolveFilePatterns =
   if (patterns) {
     for (var i = 0; i < patterns.length; ++i) {
       var fileName = patterns[i];
-      var matches = glob.sync(fileName, {cwd: cwd});
+      var matches = glob.hasMagic(fileName) ? glob.sync(fileName, {cwd: cwd}) : [fileName];
```

Comment:
Yes, this would allow names like `abc.feature:4:9:11` to be passed through to protractor-cucumber-framework and on to cucumber since it doesn't contain any special glob characters.